apps/sms/test/marionette/lib/views/dialer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var View = require('./view');
+var Dialer =  require('../../../../../../apps/communications/dialer/test/marionette/lib/dialer.js');
+
+function DialerWrapper(client) {
+  this.client = client;
+}
+module.exports = DialerWrapper;
+
+DialerWrapper.prototype = {
+  __proto__: View.prototype,
```

Comment:
Mmm, but here you should not repeat ```prototype``` as well:
```
DW.prototype = Object.create(View.prototype, {
  method1: () => {},
  method2: () => {}
});
```

Also don't you need to use ```View.apply``` in DialerWrapper "constrcutor"?